j2cl-maven-plugin/src/it/shared-lib-reactor/pom.xml

```diff
@@ -0,0 +1,28 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>shared-lib-reactor</groupId>
+    <artifactId>shared-lib-reactor</artifactId>
+    <version>1.0</version>
+    <packaging>pom</packaging>
+
+    <properties>
+        <elemental2.version>1.1.0</elemental2.version>
+        <!-- default would be Java 5, which does not work with mvn/j2cl/closure -->
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+    </properties>
+
+    <modules>
+        <module>lib</module>
+        <module>app</module>
+    </modules>
+
+    <repositories>
```

Comment:
unnecessary

Reply:
This would be our only reactor test with a pure Java shared lib -- which is my main project setup (just 30 libs instead of one). I removed the "reactor" part from the "externalizable" test, but would like to keep this one reactor-ized.